src/core/initialize.cpp

```diff
@@ -219,6 +219,19 @@ void on_integration_start() {
   check_global_consistency();
 #endif
 
+#ifdef ADDITIONAL_CHECKS
+#ifdef ELECTROSTATICS
+  if (!Utils::Mpi::all_compare(comm_cart,coulomb.method))
+    runtimeErrorMsg() << "Nodes disagree about Coulomb long range method";
```

Comment:
I'd prefer an exception here, if this condition fails there is almost certainly going to be a MPI deadlock. Alternatively we could check the runtime errors on the end of the function (if ADDITIONAL_CHECKS).